src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -456,6 +439,11 @@ private CatalogObjectRevisionEntity buildCatalogObjectRevisionEntity(final Strin
         //the tag comma separated string is synchronized in the following order tags > metadataTags > workflowXmlTags
         String synchronizedTags = synchronizeMetadataValue(tags, metadataTags, workflowXmlTags);
 
+        if (commitMessage.equals("The project name or/and tags metadata are updated")) {
```

Comment:
ouch, is there a better way to do this ?

Reply:
I know it is dirty... I can add an additional parameter (boolean) to consider the update situation.

Reply:
yeah, that would be preferable :-)